src/services/rfqm_service.ts

```diff
@@ -481,6 +488,13 @@ export class RfqmService {
             logger.error({ error }, 'Current gas price is unable to be fetched, marking worker as not ready.');
             return false;
         }
+
+        // check for outstanding jobs from the worker and resolve them
+        const unresolvedJobs = await this._dbUtils.findUnresolvedJobsAsync(workerAddress);
+        for (const job of unresolvedJobs) {
```

Comment:
can we add some logging that informs us when unresolved jobs are found?

Reply:
👍